TruthTables/Tasks.qs

```diff
@@ -44,19 +44,23 @@ namespace Quantum.Kata.TruthTables {
 
     // Task 1. Projective functions (elementary variables)
     //
-    // Goal: Describe the three projective functions x₁, x₂, x₃ as 3-input
-    //       functions, represented by integers.  Note that we follow the
-    //       convention that x₁ is the least-significant input.
+    // Goal: Describe the three projective functions f₁, f₂, f₃ represented
+    // by integers. They are 3 input functions i.e. f(x) : {0,1}ⁿ → {0,1},
+    // n being equal to 3.
     //
-    // Example: The function x₁ (least-significant input) is given as an
-    //          example.  The function is true for assignments 001, 011, 101,
-    //          and 111.
+    // f₁ should be true for the following assignments : 001, 011, 101 and 111.
+    // f₂ should be true for the following assignments : 010, 011, 110 and 111.
+    // f₃ should be true for the following assignments : 100, 101, 110 and 111.
```

Comment:
This gives away too much information (see also my comment in https://github.com/microsoft/QuantumKatas/issues/522#issuecomment-706050763)